packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -86,7 +86,7 @@ interface SdkTypeBase {
 
 export type SdkType =
   | SdkBuiltInType
-  | SdkDatetimeType
+  | SdkDateTimeType
```

Comment:
This is a breaking change as we use this type in [our emitter ](https://github.com/Azure/autorest.java/blob/main/typespec-extension/src/code-model-builder.ts#L51)